.appveyor.yml

```diff
@@ -0,0 +1,83 @@
+version: 1.0.{build}
+
+# MySQL fails to link on 2015
+image: Visual Studio 2013
+
+branches:
+  only:
+    - master
+    - appveyor_nonsense
+
+services:
+  - postgresql95
+  - mysql
+
+cache:
+  - C:\Users\appveyor\.cargo
+
+install:
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init -yv --default-toolchain stable --default-host %target%
+  - set PATH=%PATH%;%USERPROFILE%\.cargo\bin
+  - rustc -vV
+  - cargo -vV
+  - appveyor DownloadFile https://sqlite.org/2017/sqlite-dll-win64-x64-3160200.zip -FileName sqlite3.zip
+  - 7z e sqlite3.zip -y
+  - set SQLITE3_LIB_DIR=%APPVEYOR_BUILD_FOLDER%
+build: false
```

Comment:
[Their reference](https://www.appveyor.com/docs/appveyor-yml/) says it means "disable automatic builds" but… what are "automatic builds" in this context? Is this about creating artifacts?